### DISCUSSION TITLE
[Suggestion] 'Content' field

### DISCUSSION THREAD

**Discussion by Steve-JG** [verbb/formie#2070] [Ideas]
Would it be possible to include a new field type allowing for general text content to be added, one that comes with basic formatting options e.g. **B**/**I**/**U**, un/ordered lists etc?

We have a 'Heading' and 'Section' fields to group content together, but no field to perhaps add some information to briefly explain the section that has been grouped.

I am aware we have the HTML field - but this request is more with my clients in mind who are not so tech-savvy.

---

At least one other form plugin allows for general text content to be added - however I much prefer Formie :)

**Comment by engram-design** [comment]
Having a Rich Text/HTML (and maybe Twig) field is on the roadmap.

https://github.com/verbb/formie/issues/1028 and https://github.com/verbb/formie/issues/1709

**Comment by engram-design** [comment]
Added in [4.0.0-beta.5](https://github.com/verbb/formie/releases/tag/4.0.0-beta.5).
